### DISCUSSION TITLE
After upload my file to S3, can I just delete my files exist in my server?

### DISCUSSION THREAD

**Discussion by Isaw-w** [mastodon/mastodon#18448] [Q&A] [answered]
Hi, guys this is helo from Liker.Social,  recently I've decided  upload my Mastodon's media file to S3, I am not sure if I can delete these files after I uploaded? Thanks for your help.

**Comment by Gargron** [comment, marked as answer]
When you've moved all uploads to S3 and your server is configured to use S3, you can safely delete the local copies.

**Comment by Isaw-w** [comment]
Thanks, Eugen.

**Comment by jskinne3** [comment]
Could we please specify which directories are considered safe to delete after moving media storage to S3?
